v7.h

```diff
@@ -52,7 +52,19 @@ v7_val_t v7_create_null(void);
 v7_val_t v7_create_undefined(void);
 v7_val_t v7_create_string(struct v7 *v7, const char *, size_t, int);
 
+void *v7_to_foreign(v7_val_t);
+int v7_to_boolean(v7_val_t);
+double v7_to_double(v7_val_t);
+v7_cfunction_t v7_to_cfunction(v7_val_t);
+const char *v7_to_string(struct v7 *, v7_val_t *, size_t *);
+
 v7_val_t v7_get_global_object(struct v7 *);
+
+/*
+ * TODO(lsm): rename either to v7_get_property(), or
+ * rename v7_set_property() to v7_set()
```

Comment:
IMHO the user sometimes needs just to update a property value without actually thinking about the property in terms of it's attributes.

Hence, I see a `v7_set` function which will set the value of a property, by keeping its attributes unchanged (or create one with default attributes) and a `v7_define_property` which will actually allow the user to set the property attributes analogous to the JS `Object.defineProperty` function.

Since `v7_get_property` returns a pointer to an VM internal structure, we could offer a C helper analogous to the `Object.getOwnPropertyDescriptor`.